src/test/java/se/lth/cs/connect/ITUserAPI.java

```diff
@@ -188,6 +188,12 @@ public void testDeleteEmptyCollections() throws JsonParseException, UnsupportedE
 		jp = res.jsonPath();
 		String entryID = jp.getString("id");
 		
+		//Simple check to confirm that entry was created.
```

Comment:
Maybe not, it's just a check to cover that the entry always is created when the status code is 200. I thought maybe it would help with potential bugs in the future. But I can remove it.